plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -535,6 +536,40 @@ public void testAddColumnWithCommentSpecialCharacter(String comment)
         }
     }
 
+    @Test
+    public void testAddColumnWithDecimal()
+    {
+        testAddColumnWithDecimal(DecimalType.createDecimalType(14, 5));
+        testAddColumnWithDecimal(DecimalType.createDecimalType(35, 5));
+    }
+
+    protected void testAddColumnWithDecimal(DecimalType decimal)
```

Comment:
No need to create this helper method. We can simply add two columns to a table.

Reply:
done~